client/shade/pom.xml

```diff
@@ -279,6 +279,13 @@
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
                                     <manifestEntries>
                                         <Multi-Release>true</Multi-Release>
+                                        <Bundle-ManifestVersion>2</Bundle-ManifestVersion>
+                                        <Bundle-SymbolicName>org.jboss.client</Bundle-SymbolicName>
+                                        <Bundle-Version>1.0</Bundle-Version>
+                                        <Bundle-Name>OSGI Header</Bundle-Name>
```

Comment:
Hello @moulalis , it should be a name describing what is the bundle content.  `OSGI Header` still doesn't say too much.

Reply:
Hi @yersan,
I've changed it to 'JBoss Client library', if not suitable please suggest.
Thanks.